src/main/java/org/iot/dsa/dslink/dynamodb/MainNode.java

```diff
@@ -0,0 +1,58 @@
+package org.iot.dsa.dslink.dynamodb;
+
+import org.iot.dsa.dslink.jdbc.AbstractMainNode;
+import org.iot.dsa.node.DSInfo;
+import org.iot.dsa.node.DSMap;
+import org.iot.dsa.node.DSValueType;
+import org.iot.dsa.node.action.ActionInvocation;
+import org.iot.dsa.node.action.ActionResult;
+import org.iot.dsa.node.action.DSAction;
+import org.iot.dsa.util.DSException;
+
+public class MainNode extends AbstractMainNode {
+
+    ///////////////////////////////////////////////////////////////////////////
+    // Protected Methods
+    ///////////////////////////////////////////////////////////////////////////
+
+    @Override
+    protected DSAction makeAddDatabaseAction() {
+        DSAction act = new DSAction.Parameterless() {
+            @Override
+            public ActionResult invoke(DSInfo info, ActionInvocation invocation) {
+                ((MainNode) info.get()).connectDynamoDB(invocation.getParameters());
+                return null;
+            }
+        };
+        act.addParameter(Constants.CONNECTIONNAME, DSValueType.STRING, "AWS Access Key");
+        act.addParameter(Constants.ACCESSKEY, DSValueType.STRING, "AWS Access Key");
+        act.addParameter(Constants.SECRETKEY, DSValueType.STRING, "AWS Secret Key");
+        act.addParameter(Constants.REGION, DSValueType.STRING, "Region");
```

Comment:
It would make sense to have region be a drop-down (using com.amazonaws.regions.Regions.values())

Reply:
Done!